lib/public/Model.js

```diff
@@ -61,6 +61,8 @@ export default class Model extends Observable {
             case 'entry':
                 this.logs.fetchOneLog(this.router.params.id);
                 break;
+            case 'create':
```

Comment:
Using `create` is too generic as in the (short-term) future more create pages will be required.

Reply:
Renamed it to create-log-entry